src/LiveApi.js

```diff
@@ -81,7 +83,7 @@ export default class LiveApi {
     }
 
     connect(connection: ?WebSocket): void {
-        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}`;
+        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}${this.brand ? `&brand=${this.brand}` : ''}`;
```

Comment:
I can confirm that `&brand=` works fine with current unit tests, but we should check the backend behavior.